packages/flutter/test/rendering/view_chrome_style_test.dart

```diff
@@ -0,0 +1,255 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('SystemChrome - style', () {
+    const double _statusBarHeight = 25.0;
+    const double _navigationBarHeight = 54.0;
+    const double _deviceHeight = 960.0;
+    const double _deviceWidth = 480.0;
+    const double _devicePixelRatio = 2.0;
+
+    void setupTestDevice() {
+      final TestWidgetsFlutterBinding binding =
+          TestWidgetsFlutterBinding.ensureInitialized()
+              as TestWidgetsFlutterBinding;
+      const MockWindowPadding padding = MockWindowPadding(
+        top: _statusBarHeight * _devicePixelRatio,
+        bottom: _navigationBarHeight * _devicePixelRatio,
+      );
+
+      binding.window
+        ..viewPaddingTestValue = padding
+        ..paddingTestValue = padding
+        ..devicePixelRatioTestValue = _devicePixelRatio
+        ..physicalSizeTestValue = const Size(
+          _deviceWidth * _devicePixelRatio,
+          _deviceHeight * _devicePixelRatio,
+        );
+    }
+
+    tearDown(() async {
+      SystemChrome.setSystemUIOverlayStyle(const SystemUiOverlayStyle());
```

Comment:
Not sure if this is necessary, doesn't each `testWidgets` get its own binding/render tree? @goderbauer

Reply:
@jonahwilliams Looks like it's necessary, `testWidgets` doesn't clear those values by itself.